frontend/src/main/components/PersonalSchedules/PersonalScheduleForm.js

```diff
@@ -29,11 +29,8 @@ function PersonalScheduleForm({
 
   const navigate = useNavigate();
 
-  const [quarter, setQuarter] = useState(
-    {
-      quarters: quarters,
-    }.quarters[quarters.length - 1],
-  );
+  // Stryker disable next-line all : TODO: Fix the state handling for quarters
```

Comment:
This is a really tricky spot.   We need to fix this, but it deserves a PR of it's own focused on just this.